apps/sharedtest/test/unit/crop_resize_rotate_test.js

```diff
@@ -18,6 +100,10 @@ function runCropResizeRotateTests(imageWidth, imageHeight) {
     const W = imageWidth, H = imageHeight;  // The size of the test image
 
     suiteSetup(function(done) {
+
+      // We need the mock lazy loader for jpeg-exif.
+      window.LazyLoader = window.MockLazyLoader;
```

Comment:
You're going to have to make the exif stuff optional in cropResizeRotate because we don't want to do when creating thumbnails, for example. And if you do that, then you may not need to modify this test.